kore.maude

```diff
@@ -220,18 +227,30 @@ Questions about this definition:
 - do we really need ~> as a K construct?  Can it be a label and thus the way K
   deal with evaluation contexts just a particular methodology and not an
   intrisic part of K ? Same question for =>.
+  Cosmin: Syntacticly at least, I would say "no" for both. From the implementation
```

Comment:
When you say "no for both" you mean that we should NOT keep any of them as constructors, that is, that we should just replace them with particular labels?  Since => actually has a meaning in K, I am tempted to at least keep this one as a constructor.  Also, if we make `~>` into a KLabel, then there is no point in keeping `.K` as a constructor, either.  In fact, the two act together as what it called a "monad" in the denotational semantics world.  In some sense, when one thinks of K, one also thinks of these two constructs; they occur in almost all definitions, repeatedly.